src/mip/bound_strengthening.jl

```diff
@@ -0,0 +1,66 @@
+function bound_strengthening!(ps::PresolveData{T}, j::Int, ϵ::T=eps(T), ϵ_int::T=eps(T)) where{T}
```

Comment:
Can you please add a docstring describing what the bound strengthening operation is mathematically?